reporter/otlp_reporter.go

```diff
@@ -146,16 +146,13 @@ func (r *OTLPReporter) SupportsReportTraceEvent() bool { return true }
 
 // ReportTraceEvent enqueues reported trace events for the OTLP reporter.
 func (r *OTLPReporter) ReportTraceEvent(trace *libpf.Trace,
-	timestamp libpf.UnixTime64, comm, podName,
-	containerName, apmServiceName string) {
+	timestamp libpf.UnixTime64, comm, apmServiceName string, _ util.PID) {
```

Comment:
Should we remove `comm` and `apmServiceName` from the function signature in a future PR?

Reply:
My personal recommendation is not to remove `comm` and `apmServiceName` as these two arguments are provided by the eBPF programs.